platform/lang-impl/src/com/intellij/execution/runners/AbstractConsoleRunnerWithHistory.java

```diff
@@ -147,29 +141,7 @@ protected Icon getConsoleIcon() {
   }
 
   protected String constructConsoleTitle(final @NotNull String consoleTitle) {
-    if (shouldAddNumberToTitle()) {
```

Comment:
Why have you move such a small and not important function to a separate class? Are there any reasons for that?